qemu/tests/cfg/numa_opts.cfg

```diff
@@ -8,6 +8,7 @@
   kill_vm = yes
   kill_vm_gracefully = no
   encode_video_files = no
+  pre_command = "sync && echo 3 >/proc/sys/vm/drop_caches"
```

Comment:
Or we might use `utils_memory.drop_caches()`

Reply:
Yes, we could, but then we need update in several py. I would prefer to keep it is, thanks.